docs/root/intro/arch_overview/load_balancing.rst

```diff
@@ -162,12 +162,12 @@ So, level 0 endpoints will continue to receive all traffic until less than ~71.4
 healthy.
 
 The priority level logic works with integer health scores. The health score of a level is
-(level's percent of healthy endpoints) * (overprovisioning factor), capped at 100. When the system
-is relatively healthy, P=0 endpoints receive (level 0's health score) percent of the traffic, with
-the rest flowing to P=1 (assuming P=1 is healthy - more on that later). For instance, when
-50% of P=0 endpoints are healthy, they will receive 50 * 1.4 = 70% of the traffic.
-The integer percents of traffic that each level receives are collectively called the system's
-"priority load". More examples (with 2 priority levels, P=1 100% healthy):
+(level's percent of healthy endpoints) * (overprovisioning factor), capped at 100%. P=0
+endpoints receive (level 0's health score) percent of the traffic, with the rest flowing
+to P=1 (assuming P=1 is healthy - more on that later). For instance, when 50% of P=0
```

Comment:
Sorry for being picky:

> (assuming P=1 is healthy ...

to

> (assuming P=1 is 100% healthy ...

Reply:
No problem; the less vague the better. Done.